docs/Automation.md

```diff
@@ -1,183 +1,196 @@
+Standard | `A11yCriteriaId` | An enum which identifies the standards documentation from which the rule was derived.
+PropertyID | `int` | In cases where the rule tests one specific UI Automation property, this contains the UI Automation property ID in question. This property is used to link elements with rule violations to relevant documentation.
+Condition | 'string' | A description of the conditions under which a rule will be evaluated.
+
+`ElementInfo` contains the following properties:
+
+**Name** | **Type** | **Description**
 ---|---|---
-CommandConstStrings.TargetProcessId | “TargetProcessId” | The string representation (in decimal format) of the process ID to be scanned. Must be set in at least one tier (Tier 1/Tier 2/Tier 3).
-CommandConstStrings.OutputPath | “OutputPath” | The file location where files from this automation session are to be written. If the target path directory does not exist, it will be created when the first output file is written to it. If the intent is for CI / CD, it is recommended that OutputPath not be set.
-CommandConstStrings.OutputFile | “OutputFile” | The specific file (without path) where scan results are to be written. If the file already exists, it will be overwritten. The file extension can (and should) be omitted. Must be set in at least one tier (Tier 1/Tier 2/Tier 3).
-CommandConstStrings.OutputFileFormat | “OutputFileFormat” | The specific format that the output file should be generated in. The files generated will use the output path and the output filename provided. The three supported values are:<ul><li> **a11yTest** will generate "\*.a11ytest" files.<li>**Sarif** = will generate "\*.sarif" files.<li>**All** will generate both an a11ytest file and a sarif file.</ul>This parameter has lower precedence than if an extension was passed in as a part of the filename. If no extension is passed in the filename and this parameter is not passed in, then files will be generated in the sarif format. May be set in at least one tier (Tier 1/Tier 2/Tier 3). *Default value = **Sarif***.
-CommandConstStrings.NoViolationPolicy | “NoViolationPolicy” | Specifies how output files should be handled if no violations are found. <ul><li>**Discard** will discard the results file.<li>**Retain** will retain the results file.</ul>*Default value = **Retain***. Valid only as a Tier 3 setting.
-CommandConstStrings.Discard  | “Discard” | Value for NoViolationPolicy.
-CommandConstStrings.Retain | “Retain” | Value for NoViolationPolicy.
-CommandConstStrings.TeamName | “TeamName” | Value for Team name in telemetry.
+Properties | `Dictionary<string, string>` | A string to string dictionary where the key is a UIAutomation property name and the value is the corresponding UI Automation property value.
+Patterns | `IEnumerable<string>` | A list of names of supported patterns.
 
 ### Using the assembly
 You can get the files via a NuGet package Configure NuGet to retrieve the
```

Comment:
A line/sentence break would be helpful ("You can get the files via a NuGet package Configure NuGet"). This wasn't from your PR but it would be nice to fix while we're here